web/wp-content/themes/dctx/search.php

```diff
@@ -8,20 +8,21 @@
  * @since 1.0.0
  */
 
-// phpcs:disable
-
 get_header();
+$search_result = get_query_var( 's', 1 );
 ?>
 
 <div id="main">
-	<h1 class="page-title">Search results for '<?php echo $_GET['s']; ?>'</h1>
+	<h1 class="page-title">Search results for '<?php echo $search_result ); ?>'</h1>
```

Comment:
:no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$search_result' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).